django/santropolFeast/member/views.py

```diff
@@ -632,6 +632,34 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class ClientStatusView(generic.DetailView):
+    model = Client
+    template_name = 'client/view/status.html'
+
+    # DON'T KNOW WHAT THIS METHOD IS USED FOR
+    # STATUS PAGE IS OK WITH OR WITHOUT..
+    # @method_decorator(login_required)
+    # def dispatch(self, *args, **kwargs):
+    #     return super(ClientStatusView, self).dispatch(*args, **kwargs)
+
```

Comment:
`@method_decorator(login_required)` will check that the user is logged to be able to see the page, so if the URL should be visible to logged users only, we need to use this.